### ISSUE TITLE
Pre- and/or post-project impact appraisals

### ISSUE THREAD

**Issue by publishwhatyoufund** [pwyf/2022-Index-indicator-definitions#32]
## Description

Pre-project impact appraisals explain the totality of positive and negative, primary and secondary effects expected to be produced by a development intervention.

This is only expected if the activity is in the implementation, completion or post-completion phase.

This is not expected if the default aid type code is administrative costs (G01).

## Proposed test

```python
 For each current activity,
if `activity-status/@code` is one of (2, 3, 4)
And `default-aid-type/@code` is not any of A01, A02, B01, B02, B03, B04, D01, D02, E01, E02, F01, H01, H02, H03, H04, H05 or G01
    And `transaction/aid-type/@code` is not any of A01, A02, B01, B02, B03, B04, D01, D02, E01, E02, F01, H01, H02, H03, H04, H05 or G01
then `document-link/category[@code="A01"]` should be present

```

**Comment by breidertmt** [comment]
If there is a policy document on the organization's determination of when these impact appraisals are required and when they are not per the organization, how should that be coded? Should it be coded as a document category A01?

**Comment by AndieVaughn** [comment]
USAID supports including impact appraisals but limiting the number of sectors where these are attributed.  Realistically, these may not be completed in specific sectors for practical or policy reasons.

**Comment by ForeignCommonwealthDevelopmentOffice** [comment]
Please be clear on the definition of impact appraisal that you will apply for sampling, distinguishing between the type of appraisal that might be required for an infrastructure investment and that for a programme in a different sector (eg. TA).  Also FCDO agrees with USAID, in some countries or some sectors we may not publish these documents, for example due to security reasons.

**Comment by jzheng-79** [comment]
UNICEF supports including impact appraisals but requests to consider/accept documents in various formats, other than the typical project-type appraisal document.

As UNICEF already adopted results-based model, information similar to pre-project impact appraisals is available in various documents, such as Programme Strategy Notes and Country Programme Action Plans, where the theory of change (pathway to achieving results), potential risk, assumptions, and impact of the planned development interventions of the Country Programme are detailed. 

The definition of the indicator could be expanded to country level, as an intervention rarely has isolated impact.

**Comment by publishwhatyoufund** [comment]
**@breidertmt**: Yes, they should be coded using the A01 document code for the activity. 

**@AndieVaughn**: We have limited this test significantly – we excluded all aid types C01 after the last consultation for the 2020 Index.

**@ForeignCommonwealthDevelopmentOffice, @jzheng-79**: for this indicator any pre-project impact appraisal that meets the indicator criteria will suffice. We will also accept security redacted documents for this indicator.
